tutorcodejail/plugin.py

```diff
@@ -17,7 +17,7 @@
         "VERSION": __version__,
         "HOST": "codejailservice",
         "DOCKER_IMAGE": f"docker.io/ednxops/codejailservice:{__version__}",
-        "SANDBOX_PYTHON_VERSION": "3.5.10",
```

Comment:
We changed it from 3.5 to 3.8 since the sandbox requirements didn't include py3.5 anymore. 
Now, which patch version should we use here? 
We first tried to use 3.8.6 but then changed it to 3.8.5 so it doesn't crash with the codejailservice pyenv (which also uses 3.8.6).
@felipemontoya @Alec4r

Reply:
maybe you can use [pyenv-alias](https://github.com/s1341/pyenv-alias) and install the sandbox version as `x.y.z_sandbox` ?

Reply:
That's a great suggestion! Thanks, I'll try that